lib/block-supports/block-style-variations.php

```diff
@@ -79,6 +79,38 @@ function gutenberg_render_block_style_variation_support_styles( $parsed_block )
 		return $parsed_block;
 	}
 
+	// Recursively resolve any ref values with the appropriate value within the
+	// theme_json data.
+	$replace_refs = function ( &$variation_data ) use( &$replace_refs, $theme_json ) {
```

Comment:
After WP6.6 I'd like to try and clean up some of this block support's code. To that end, I've tried to avoid introducing another global function that would then need to be deprecated.